bin/crew

```diff
@@ -1274,10 +1274,10 @@ end
 
 def install_files(src, dst = File.join( CREW_PREFIX, src.delete_prefix('./usr/local') ))
   if Dir.exist?(src)
-    if File.executable?("#{CREW_PREFIX}/bin/crew-mvdir") && !CREW_DISABLE_MVDIR
+    if File.executable?("#{CREW_PREFIX}/bin/crew-mvdir") && !CREW_DISABLE_MVDIR && CREW_ENABLE_MVDIR
```

Comment:
Why do we need `CREW_DISABLE_MVDIR && CREW_ENABLE_MVDIR`?  One or the other but not both?

Reply:
I took a light touch to the existing code, to only add the workaround... that way it can be removed later once this is fixed.... as we want to keep using crew-mvdir as the default.

Reply:
Not quite understanding.  Does the original constant not work?